addons/mail/wizard/mail_template_preview_view.xml

```diff
@@ -2,16 +2,24 @@
 <odoo>
     <data>
         <!-- Email Template Preview -->
-        <record model="ir.ui.view" id="email_template_preview_form">
-            <field name="name">email_template.preview.form</field>
-            <field name="model">email_template.preview</field>
+        <record model="ir.ui.view" id="mail_template_preview_form">
+            <field name="name">mail.template.preview.form</field>
+            <field name="model">mail.template.preview</field>
             <field name="arch" type="xml">
                 <form string="Email Preview">
-                    <field name="model_id" invisible="1"/>
-                    <h3>Preview of <field name="name" readonly="1" nolabel="1" class="oe_inline"/></h3>
-                    Choose an example <field name="model_id" class="oe_inline" readonly="1"/> record:
-                    <field name="res_id" class="oe_inline" style="margin-left: 8px;"/>
-                    <p>Choose a language: <field name="preview_lang" class="oe_inline ml8"/></p>
+                    <h3>Preview of <field name="mail_template_id" readonly="1" nolabel="1" class="oe_inline"/></h3>
```

Comment:
I would consider adding `options="{'no_open': True}"` on `mail_template_id` and `model_id`.

I think it's more confusing than helping for the end user, since it's very technical stuff.
What do you think?

Reply:
Thank you, I am agree for the mail_template_id to avoid looping in forms. But I think the model_id we can keep like, this model is already a technical one for now (we can't access it only in the debug mode). Also, the model is clickable on the form of the mail template form :)

Reply:
You are right, my bad, there is something buggy here. I am on it